README.md

```diff
@@ -3,66 +3,79 @@ kindle-highlights
-    "embeddedId"    => "From_the_Garden_to_the_City:420E805A",
-    "endLocation"   => 54220,
-    "highlight"     => "While God's words are eternal and unchanging, the tools we use to access those words do change, and those changes in technology also bring subtle changes to the practice of worship. When we fail to recognize the impact of such technological change, we run the risk of allowing our tools to dictate our methods. Technology should not dictate our values or our methods. Rather, we must use technology out of our convictions and values.",
-    "howLongAgo"    => "1 year ago",
-    "startLocation" => 53780,
-    "timestamp"     => 1321038422000
+    "B002JCSCO8" => "The Art of the Commonplace: The Agrarian Essays of Wendell Berry",
+    "B0049SPHC0" => "Calvinistic Concept of Culture, The",
+    "B000JMKZX6" => "The Essays of Arthur Schopenhauer; On Human Nature",
+    "B005CQ2ZE6" => "From the Garden to the City",
+    "B0082ZJFCO" => "The Golden Sayings of Epictetus",
+    "B000SEGEKI" => "The Pragmatic Programmer: From Journeyman to Master",
+    "B009D6AGOM" => "The Rare Jewel of Christian Contentment",
+    "B00E25KVLW" => "Ruby on Rails 4.0 Guide",
+    "B004X5RLBY" => "The Seven Lamps of Architecture",
+    "B0032UWX1O" => "The Westminster Confession of Faith",
+    "B0026772N8" => "Zen and the Art of Motorcycle Maintenance"
   }
-]
-```
+    ```
+1. (Optional) If you are not sure how to produce human-readable output as in the preceding example, refer to [Best way to pretty print a hash]( http://stackoverflow.com/questions/8842546/best-way-to-pretty-print-a-hash).
+1. Get your highlights for a specific book by using its ASIN:
+
+  `kindle.highlights_for("B005CQ2ZE6")`
+  
+  Example output is as follows:
+
+  ```
+  [
+    {
+      "asin"          => "B005CQ2ZE6",
+      "customerId"    => "...",
+      "embeddedId"    => "From_the_Garden_to_the_City:420E805A",
+      "endLocation"   => 29591,
+      "highlight"     => "One of the most dangerous things you can believe in this world is that technology is neutral.",
+      "howLongAgo"    => "1 year ago",
+      "startLocation" => 29496,
+      "timestamp"     => 1320901233000
+    },
+    {
+      "asin"          => "B005CQ2ZE6",
+      "customerId"    => "...",
+      "embeddedId"    => "From_the_Garden_to_the_City:420E805A",
+      "endLocation"   => 54220,
+      "highlight"     => "While God's words are eternal and unchanging, the tools we use to access those words do change, and those changes in technology also bring subtle changes to the practice of worship. When we fail to recognize the impact of such technological change, we run the risk of allowing our tools to dictate our methods. Technology should not dictate our values or our methods. Rather, we must use technology out of our convictions and values.",
+      "howLongAgo"    => "1 year ago",
+      "startLocation" => 53780,
+      "timestamp"     => 1321038422000
+    }
+  ]
+  ```
 
-### In The Wild
+### Alternatives
```

Comment:
This list collects projects using `kindle-highlights`; it’s not a list of alternatives.